modules/flowable-engine/src/main/java/org/flowable/engine/impl/history/DefaultHistoryManager.java

```diff
@@ -308,6 +308,16 @@ public void recordTaskInfoChange(TaskEntity taskEntity, String activityInstanceI
         }
     }
 
+
+    @Override
+    public void recordHistoricTaskDeleted(String taskId){
+        HistoricTaskInstanceEntity historicTaskInstance = processEngineConfiguration.getTaskServiceConfiguration().getHistoricTaskService().getHistoricTask(taskId);
+
+        if (historicTaskInstance != null && hasTaskHistoryLevel(historicTaskInstance.getProcessDefinitionId())) {
```

Comment:
It appears that `hasTaskHistoryLevel` cannot be found:
<pre>
Error:  /home/runner/work/flowable-engine/flowable-engine/modules/flowable-engine/src/main/java/org/flowable/engine/impl/history/DefaultHistoryManager.java:[303,45] cannot find symbol
  symbol:   method hasTaskHistoryLevel(java.lang.String)
  location: class org.flowable.engine.impl.history.DefaultHistoryManager
Error:  /home/runner/work/flowable-engine/flowable-engine/modules/flowable-engine/src/main/java/org/flowable/engine/impl/history/async/AsyncHistoryManager.java:[287,45] cannot find symbol
  symbol:   method hasTaskHistoryLevel(java.lang.String)
  location: class org.flowable.engine.impl.history.async.AsyncHistoryManager
[INFO] 2 errors 
</pre>